.github/workflows/custom_docker_builds.yml

```diff
@@ -42,7 +42,7 @@ jobs:
           - docker-image: ./analytics
             image-tags: ghcr.io/spack/django:0.3.5
           - docker-image: ./images/ci-prune-buildcache
-            image-tags: ghcr.io/spack/ci-prune-buildcache:0.0.3
+            image-tags: ghcr.io/spack/ci-prune-buildcache:0.0.4
```

Comment:
Doesn't changing the tag here mean you need to change it in the `cronjobs.yaml` as well?

Reply:
oh shoot, yeah, thanks!